src/curated-corpus/components/SuggestedScheduleItemListCard/SuggestedScheduleItemListCard.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from 'react';
```

Comment:
do you think we should stay consistent and move other component tests into their own tests rather than testing all components here?

Reply:
I could, yes. This feels more integrated but yeah I can refactor that!

Reply:
Done!